frameworks/JavaScript/ringojs/app/models.js

```diff
@@ -1,26 +1,18 @@
-var {Store, Cache} = require('ringo-sqlstore');
-
-// DO NOT TOUCH THE FOLLOWING LINE.
-// THIS VARIABLE IS REGEX REPLACED BY setup.py
-var dbHost = 'TFB-database';
+const { Store, Cache } = require('ringo-sqlstore');
 
 // create and configure store
-var connectionPool = module.singleton("connectionPool", function() {
-    var mysqlConnectionProperties = "?jdbcCompliantTruncation=false&elideSetAutoCommits=true&useLocalSessionState=true&cachePrepStmts=true&cacheCallableStmts=true&alwaysSendSetIsolation=false&prepStmtCacheSize=4096&cacheServerConfiguration=true&prepStmtCacheSqlLimit=2048&zeroDateTimeBehavior=convertToNull&traceProtocol=false&useServerPrepStmts&enableQueryTimeouts=false&useUnbufferedIO=false&useReadAheadInput=false&maintainTimeStats=false&cacheRSMetadata=true";
+const connectionPool = module.singleton("connectionPool", function () {
+    const mysqlConnectionProperties = "?jdbcCompliantTruncation=false&elideSetAutoCommits=true&useLocalSessionState=true&cachePrepStmts=true&cacheCallableStmts=true&alwaysSendSetIsolation=false&prepStmtCacheSize=4096&cacheServerConfiguration=true&prepStmtCacheSqlLimit=2048&zeroDateTimeBehavior=convertToNull&traceProtocol=false&useServerPrepStmts&enableQueryTimeouts=false&useUnbufferedIO=false&useReadAheadInput=false&maintainTimeStats=false&cacheRSMetadata=true";
     return Store.initConnectionPool({
-        "url": "jdbc:mysql://" + dbHost + "/hello_world" + mysqlConnectionProperties,
+        "url": "jdbc:mysql://TFB-database/hello_world" + mysqlConnectionProperties,
         "driver": "com.mysql.jdbc.Driver",
         "username": "benchmarkdbuser",
         "password": "benchmarkdbpass",
         "minimumIdle": 10,
         "maximumPoolSize": 30
     });
 });
-var store = exports.store = new Store(connectionPool);
-var queryCache = module.singleton("queryCache", function() {
-    return new Cache(10000);
-});
-store.setQueryCache(queryCache);
```

Comment:
Please don't remove the query cache. It only caches parsed queries, not database results (see https://github.com/grob/ringo-sqlstore/wiki/Querycache for an explanation).